app/models/message.rb

```diff
@@ -0,0 +1,2 @@
+class Message < ApplicationRecord
+end
```

Comment:
please add the association

Reply:
Added the required associations on the last commit